go.sum

```diff
@@ -1,12 +1,26 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
+cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6AU=
+cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
+cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
+cloud.google.com/go v0.46.3/go.mod h1:a6bKKbmY7er1mI7TEI4lsAkts/mkhTSZK8w33B4RAg0=
+cloud.google.com/go/bigquery v1.0.1/go.mod h1:i/xbL2UlR5RvWAURpBYZTtm/cXjCha9lbfbpx4poX+o=
```

Comment:
I wonder why are these needed? :confused: I checked out your branch and ran this: 
```
$ go mod why cloud.google.com/go/bigquery
# cloud.google.com/go/bigquery
(main module does not need package cloud.google.com/go/bigquery)
$ go mod why cloud.google.com/go         
# cloud.google.com/go
(main module does not need package cloud.google.com/go)
```

:confused:

Reply:
I think I forgot to mention this .. but it is :drum: https://github.com/spf13/viper , which imports most things IMO:
```
$ go mod why -m cloud.google.com/go
# cloud.google.com/go
github.com/spf13/viper/remote
github.com/bketelsen/crypt/config
github.com/bketelsen/crypt/backend/firestore
cloud.google.com/go/firestore
cloud.google.com/go
```

The good think is that go.sum is just the list of all modules and their versions that anything in the dependency list has sait it needs not stuff that we actually add to vendor or compile into the binary ... so this is mostly noise

Reply:
https://pkg.go.dev/github.com/spf13/viper#readme-remote-key-value-store-support :man_facepalming: 
Have I mentioned how much I ":heart:" Go modules recently?